src/web/app/pages-instructor/instructor-session-edit-page/instructor-session-edit-page.component.ts

```diff
@@ -252,18 +252,19 @@ export class InstructorSessionEditPageComponent extends InstructorSessionBasePag
     // load course candidates first
     this.sessionEditFormModel.isCopying = true;
     this.courseService.getInstructorCoursesThatAreActive()
+    .pipe(finalize(() => this.sessionEditFormModel.isCopying = false))
     .subscribe((courses: Courses) => {
       const modalRef: NgbModalRef = this.ngbModal.open(CopySessionModalComponent);
       modalRef.componentInstance.newFeedbackSessionName = this.feedbackSessionName;
       modalRef.componentInstance.courseCandidates = courses.courses;
       modalRef.componentInstance.sessionToCopyCourseId = this.courseId;
 
       modalRef.result.then((result: CopySessionModalResult) => {
-        this.failedToCopySessions = {};
```

Comment:
Have you tried doing the following:
- Make a failed copy
- Make a successful copy
I believe this line is to remove the errors from previous actions

Reply:
oh ya, I just tested it. Yup, I will revert it back